lib/DDG/Spice/Thumbtack.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Thumbtack;
+# ABSTRACT: Returns service providers nearby
+
+use DDG::Spice;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
+spice is_cached => 1;
+
+name "Thumbtack Search";
+source "Thumbtack";
+icon_url "https://www.thumbtack.com/favicon.ico";
+description "Shows a list of service providers";
+primary_example_queries "plumbers near me", "electricians close by";
+category "location_aware";
+topics "everyday";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Thumbtack.pm";
+attribution github => ["http://github.com/whalenrp", "whalenrp"];
+
+spice to => 'https://www.thumbtack.com/search/$1/?callback={{callback}}&state=$2&city=$3';
+spice from => '(.*)/(.*)/(.*)';
+
+triggers end => 'near me', 'around me', 'close by';
+triggers startend => 'nearby';
```

Comment:
I'm concerned this is going to greatly over-trigger, sending you a ton of unnecessary API calls. It would probably be easier to detect queries that have "plumber", "electrician", etc.

Would you be able to provide a list of the top...50, 100, or so types of professionals? We can put them in a text file and load them in as a list of trigger words.

Then verify the query matches a pattern we expect "X nearby", "nearby X", etc

The Quandl IA is a great example of this, there are many more in the repo

https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Quandl/WorldBank.pm#L20